sklearn/utils/tests/test_class_weight.py

```diff
@@ -282,3 +283,9 @@ def test_class_weight_does_not_contains_more_classses():
 
     # Does not raise
     tree.fit([[0, 0, 1], [1, 0, 1], [1, 2, 0]], [0, 0, 1])
+
+
+def test_compute_sample_weight_sparse():
+    y = sparse.csc_matrix(np.asarray([0, 0, 0, 1, 1, 1])).T
```

Comment:
```suggestion
    """Check that we can compute weight for sparse `y`."""
    y = sparse.csc_matrix(np.asarray([0, 0, 0, 1, 1, 1])).T
```

Reply:
Could we take an example where we have twice more 1 than 0 to be sure that the computation is not returning only ones?